serialization/etf/src/main/java/net/automatalib/serialization/etf/writer/Mealy2ETFWriterAlternating.java

```diff
@@ -0,0 +1,177 @@
+/* Copyright (C) 2013-2017 TU Dortmund
+ * This file is part of AutomataLib, http://www.automatalib.net/.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.automatalib.serialization.etf.writer;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import net.automatalib.automata.transout.MealyMachine;
+import net.automatalib.words.Alphabet;
+import org.apache.commons.lang3.tuple.Triple;
```

Comment:
Looking at the code I am not using `getFirst()`, `getSecond()` etc. So the reason I am using the triple is because it comes with `equals()` and `hashcode()`. If I were to create an inner class I would add a dependency on lombok for `equalsAndHashcode`, so I do not think there is much to gain here.

Reply:
Technically, there is something to gain, I think. As far as I understand Lombok, it is just an annotation-processing tool, so it would only introduce a compile-time dependency (for generating the equals/hascode method) and not a runtime dependency (the Triple class).

You can leave it for now, but I will probably refactor it someday (I like your idea with Lombok, though. It may also help to cleanup our internal Pair class).